spec/controllers/admin/events_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Admin::EventsController do
+  let(:conference) { create(:conference) }
+  let(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
+  let(:organizer) { create(:user, role_ids: organizer_role.id) }
+  let(:event_without_commercial) { create(:event, program: conference.program) }
+  let(:event_with_commercial) { create(:event, program: conference.program) }
```

Comment:
This way you are not using this anywhere. Leave it with ! to be created in the beginning.